### ISSUE TITLE
Remove application validation from SignAndSubmit page

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5357]
## ❌ Current behaviour (issue)
Signature box does not allow for keyboard or voice navigation users to enter their name and submit their application if they don't realize that they have outstanding skills justifications. 

### Location 
Sign and Submit UI

### Category: 
Critical Accessiblity Bug

## AT configuration(s): 
•	JAWS, Desktop, Windows 11, Edge browser. 
•	Voice Control, Mobile, IOS, Chrome

## Reproduction info: 
With JAWS/ NVDA screen reader: 
1.	Tab through the “Sign and Submit UI to the signature box. 
2.	Hit enter to access the box. 
3.	Try to enter your name in the box. 
	
With Voice Control: 
1.	Navigate into the signature box using the item names or item numbers
2.	Choose numbered grid overlay. 
3.	Select the number for the signature box 
4.	Try to dictate the letters of your name or enter your name using a screen reader and keyboard navigation for the signature. 

### Screen Shot
![#5357_Photo.png](https://images.zenhubusercontent.com/625076b8f67c446dd188ffe5/4e70c24a-81b7-4403-812a-11533be85ac7)

From Fable video:
![image](https://user-images.githubusercontent.com/8978655/214911773-7f770a9d-f2ad-46ec-a56e-61ecba3c3273.png)

### Fable Video Clip  
Signature Interaction Issues [https://app.makeitfable.com/clip/63bcd337604482001abf7428](url)

## ✔ Expected behaviour (fix)
User can sign and submit their application. 

## **WCAG 2.1 Criteria** 
[2.1.1 Keyboard Navigation](https://www.w3.org/WAI/WCAG22/Understanding/keyboard), [3.4 Input Assistance](https://www.w3.org/WAI/WCAG22/Understanding/)

### Acceptance Criteria
- [ ] Form only validates the signature box - does not disable for other scenarios like skills missing or language missing

## Additional information (optional)
1. Frustrated AT Users cannot complete and submit their application.

**Comment by petertgiles** [comment]
@RM-1978 Could you please try to get the exact versions of the tools used to help us reproduce?

**Comment by petertgiles** [comment]
There is no hidden text in this button so probably not the same issues as #5358.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/fca285a93efbfc7dba2fc47c841c09693af27c24/apps/web/src/pages/SignAndSubmitPage/SignAndSubmitPage.tsx#L200

**Comment by petertgiles** [comment]
I noticed that this box is disabled if the user somehow gets to the page without a completed application.  Perhaps that's what happened in this Fable test?
https://github.com/GCTC-NTGC/gc-digital-talent/blob/fca285a93efbfc7dba2fc47c841c09693af27c24/apps/web/src/pages/SignAndSubmitPage/SignAndSubmitPage.tsx#L213

**Comment by RM-1978** [comment]
Not terribly certain. I know that Doug skipped engaging with the skills and experience picker in the interest of time. He may have even asked the fable testing team  fill in the required skills and their explanations.  I don't think that Hassan did though.
That said, there didn't seem to be an error notice on Hassan or Doug's submission page that would explain why the box was unavailable to them.

**Comment by petertgiles** [comment]
Yeah, there's definitely no error message on that page.  We could design some error messages for that page but I don't think it's worth it since users _shouldn't_ able to get into that state under normal use.  My opinion is we should just remove the form disable logic here since it's confusing.  Just let the user submit their application and if it is rejected they'll see the error message returned from the server.

**Comment by RM-1978** [comment]
Hi Just checked Hassan's video. I think there may have been a couple of skills he didn't enter that might have caused this., but I agree  with your solution Are you cool with that @gobyrne?

**Comment by petertgiles** [comment]
I just took another peek at the video and the submit button does look disabled (lighter coloured).  I bet there are skills missing and they just brought up the "step 2" page from browser history.
